packages/datagateway-dataview/src/views/landing/isis/isisInvestigationLanding.component.tsx

```diff
@@ -394,67 +391,16 @@ const LandingPage = (props: LandingPageProps): React.ReactElement => {
             <Typography aria-label="landing-investigation-publisher">
               {t('doi_constants.publisher.name')}
             </Typography>
-
-            <Typography
-              className={classes.subHeading}
-              component="h6"
-              variant="h6"
-              aria-label="landing-investigation-citation-label"
-            >
-              {t('studies.details.citation_label')}
-            </Typography>
-            <Typography aria-label="landing-investigation-citation_format">
-              {t('studies.details.citation_format')}
-            </Typography>
-            <Typography aria-label="landing-investigation-citation">
-              <i ref={citationRef}>
-                {formattedUsers.length > 1 &&
-                  `${formattedUsers[0].fullName} et al; `}
-                {formattedUsers.length === 1 &&
-                  `${formattedUsers[0].fullName}; `}
-                {studyInvestigation &&
-                  studyInvestigation[0]?.study?.startDate &&
-                  `${studyInvestigation[0].study.startDate.slice(0, 4)}: `}
-                {title && `${title}, `}
-                {t('doi_constants.publisher.name')}
-                {doi && ', '}
-                {doi && (
-                  <MuiLink
-                    href={`https://doi.org/${doi}`}
-                  >{`https://doi.org/${doi}`}</MuiLink>
-                )}
-              </i>
-            </Typography>
-            {!copiedCitation ? (
-              <Button
-                id="landing-investigation-copy-citation"
-                aria-label="landing-investigation-copy-citation"
-                variant="contained"
-                color="primary"
-                size="small"
-                onClick={() => {
-                  if (citationRef?.current?.textContent) {
-                    navigator.clipboard.writeText(
-                      citationRef.current.textContent
-                    );
-                    setCopiedCitation(true);
-                    setTimeout(() => setCopiedCitation(false), 1500);
-                  }
-                }}
-              >
-                Copy citation
-              </Button>
-            ) : (
-              <Button
-                id="landing-investigation-copied-citation"
-                variant="contained"
-                color="primary"
-                size="small"
-                startIcon={<Mark size={20} visible={true} />}
-              >
-                Copied citation
-              </Button>
-            )}
+            <CitationFormatter
+              doi={doi}
+              formattedUsers={formattedUsers}
+              title={title}
+              startDate={
+                studyInvestigation
+                  ? studyInvestigation[0]?.study?.startDate
```

Comment:
The study very often does not have a start date itself and instead it gets the start date from the underlying investigation. So here this should just be the investigation startDate